content/post/21-10-01-progress-report-27.md

```diff
@@ -0,0 +1,281 @@
+
+We evaluated a few options but eventually decided on self-hosted [Wiki.js](https://js.wiki/) as a platform for our docs. It's not the best but it's the best option we could find.
+
+The new documentation site is up at https://docs.spacestation14.io. We are still in the process of moving articles over and making everything look nice, but so far we're quite pleased.
+
+To be clear: these are *developer* docs. We're not planning to have a wiki for actual game content *yet*. We'd like to try an in-game help system of some form first, but nothing is set in stone yet on that regard.
+
+## Content
+
+### Pneumatic Cannons
+*Contributed by mirrorcult*
+
+Load [INSERT ITEM]. Shoot [INSERT ITEM] at people. Profit?
+
+{{< video-auto "/video/pr_27/pneumatic.mp4" >}}
+
+### Board Games
+*Contributed by Visne and Zumorica*
+
+Bored by the lack of content? Why not sit down and play a nice relaxing game of Chess! This feature is highly extensible and should be thought of like a 2D Tabletop Simulator inside of Space Station 14.
+
+{{< video-auto "/video/pr_27/chess.mp4" >}}
+
+Zumorica further improved upon Visne's code making it easy to add different tabletop games, adding the game of parchís in the process!
+
+{{< imgw "/images/post/pr_27/sorry" >}}
+
+### Alt-Clicking
+*Contributed by ElectroSR*
+
+Alt+E, Alt+Z and Alt+Left-Mouse keybindings can trigger alternative interactions on entities. Some existing context-menu actions are now considered "alternative interactions", for example ejecting IDs/beakers/magazines or locking & unlocking lockers can be done via these new keybindings.
+
+{{< imgw "/images/post/pr_27/alt" >}}
+
+### Wielding
+*Contributed by mirrorcult*
+
+You can now wield (two-hand) some weapons like the fire-axe which will increase the damage per swing. The fire-axe has also received a buff against windows and wooden objects.
+
+{{< video-auto "/video/pr_27/wielding.mp4" >}}
+
+### Ambient sounds
+*Contributed by metalgearsloth, Swept*
+
+Ambient sounds can now play from objects around the station for improved immersion. APCs hum, vending machines whirr, and so on.
+
+<!-- TODO: Align Left -->
+{{< video-audio "/video/pr_27/ambience.mp4" >}}
+
+### Vending Machine Ads
+*Contributed by Moses, Visne and Zumorica*
+
+{{< imgw "/images/post/pr_27/ads" >}}
+
+### Clothing Storage Visualizers!
+
+*Contributed by Ygg01 and Swept*
+
+Certain items when placed in belts show a sprite on the belt icon itself.
+
+<!-- TODO: Align Left -->
+{{< video-auto "/video/pr_27/belts.mp4" >}}
+
+A lot of belts have gotten sprites for this feature, as well as the botany belt being added to give botanists a helping hand.
+
+{{< imgw "/images/post/pr_27/belts" >}}
+
+### Meth
+*Contributed by scaly-chimp*
+
+Scaly-chimp broke bad and added meth so aspiring chemists can recreate scenes from the hit TV show Big Bang Theory.
+
+<!-- Insert video of someone high on meth -->
```

Comment:
*high on meth _in the game_
![image](https://user-images.githubusercontent.com/16618384/136602592-adb31961-a601-4cc0-acb3-01df72489992.png)